scripts/core/ui/components/SelectUser.tsx

```diff
@@ -72,6 +72,7 @@ export class SelectUser extends React.Component<IPropsSelectUser, IState> {
 
         return (
             <Select2
+                focus={true}
```

Comment:
Can you call it `autoFocus` as in native input fields?

Reply:
sure